orb/orb.yml

```diff
@@ -122,6 +122,11 @@ jobs:
         type: string
         default: "/tmp"
         description: Path in the workspace to store the secrets
+      vault-ending:
```

Comment:
Why not use `$CLUSTER_ENV` instead?
Then we can just remove this parameter

Reply:
If it always adds up that way im all for it

Reply:
Just wasn't sure if that would always be the case